package.json

```diff
@@ -3,28 +3,84 @@
   "module": "index.ts",
   "type": "module",
   "scripts": {
-    "start": "vite"
+    "start": "vite",
+    "build": "vite build",
+    "format": "biome format --write src tsconfig.json package.json"
   },
   "devDependencies": {
     "@biomejs/biome": "^1.9.4",
-    "@tscircuit/runframe": "^0.0.347",
+    "@tscircuit/runframe": "^0.0.1135",
     "@types/bun": "latest",
-    "@types/react": "^19.0.5",
-    "@types/react-dom": "^19.0.3",
-    "@vitejs/plugin-react": "^4.3.4",
+    "@types/react": "^19.2.2",
+    "@types/react-dom": "^19.2.2",
+    "@vitejs/plugin-react": "^4.7.0",
     "class-variance-authority": "^0.7.1",
-    "lucide-react": "^0.471.0",
-    "react": "18",
-    "react-dom": "18",
-    "vite": "^6.0.7",
+    "lucide-react": "^0.471.2",
+    "react": "^19.0.0",
+    "react-dom": "^19.0.0",
+    "vite": "^6.4.1",
     "vite-tsconfig-paths": "^5.1.4"
   },
   "peerDependencies": {
-    "typescript": "^5.0.0"
+    "typescript": "^5.9.3"
   },
   "dependencies": {
-    "@tscircuit/core": "^0.0.390",
-    "circuit-json": "^0.0.144",
-    "zustand": "^5.0.3"
+    "@radix-ui/react-alert-dialog": "^1.1.15",
+    "@radix-ui/react-checkbox": "^1.3.3",
+    "@radix-ui/react-dialog": "^1.1.15",
+    "@radix-ui/react-dropdown-menu": "^2.1.16",
+    "@radix-ui/react-icons": "^1.3.2",
+    "@radix-ui/react-popover": "^1.1.15",
+    "@radix-ui/react-progress": "^1.1.7",
+    "@radix-ui/react-slot": "^1.2.3",
+    "@radix-ui/react-tabs": "^1.1.13",
+    "@resvg/resvg-wasm": "^2.6.2",
+    "@tscircuit/3d-viewer": "^0.0.418",
+    "@tscircuit/assembly-viewer": "^0.0.5",
+    "@tscircuit/capacity-autorouter": "^0.0.133",
+    "@tscircuit/checks": "^0.0.85",
+    "@tscircuit/core": "^0.0.813",
```

Comment:
You can get rid of core and just use the `tscircuit` package. I think most of the other packages are not needed then cause `tscircuit` has it

Reply:
Are you sure @imrishabh18? `tscircuit` bundles `@tscircuit/core` but not `@tscircuit/runframe`, which we need. So we'd still need to install `@tscircuit/runframe` separately, and it has most of the same dependencies as core. On top of adding stuff that we don't need for this project. What say you @seveibar ?

Reply:
I don't think so you are right. Runframe is present in the tscircuit package, make sure you take a look at the package.json. This seem to be a very simple PR to do and it's loosing so much time in just the discussion

<img width="1762" height="1450" alt="image" src="https://github.com/user-attachments/assets/73bdcaef-95ec-42c9-9fec-53b22e1eb76f" />

Reply:
Its not wasting time to discuss. I was only asking if you are sure. Will implement

Reply:
https://github.com/tscircuit/circuitjson.com/blob/94fa2b7635fd82b5e9b54de13b5a2f11b58b0611/src/App.tsx#L3
@imrishabh18 how can we access `CircuitJsonPreview` without  `@tscircuit/runframe`? 

<img width="650" height="111" alt="Screenshot 2025-10-23 at 13 39 08" src="https://github.com/user-attachments/assets/2e80636c-ead4-4290-b1af-e947d529e75f" />

Reply:
Applied changes

Reply:
nice catch @imrishabh18